src/Uno.UWP/Graphics/Display/DisplayInformation.wasm.cs

```diff
@@ -152,6 +161,24 @@ private static DisplayOrientations ParseJsOrientation(string jsOrientation)
 				_ => DisplayOrientations.None,
 			};
 		}
+
+		private static string ToJsOrientation(DisplayOrientations orientations)
+		{
+			return orientations switch
+			{
+				DisplayOrientations.Landscape => "landscape-primary",
+				DisplayOrientations.LandscapeFlipped => "landscape-secondary",
+				DisplayOrientations.Portrait => "portrait-primary",
+				DisplayOrientations.PortraitFlipped => "portrait-secondary",
+				_ => "any"
```

Comment:
Did you find "any" in some documentation or just arbitrary string?

Reply:
It's documented [here](https://developer.mozilla.org/en-US/docs/Web/API/ScreenOrientation/lock).